### ISSUE TITLE
What is the purpose of the A11y View HTML?

### ISSUE THREAD

**Issue by zepumph** [phetsims/a11y-research#63]
In https://github.com/phetsims/chipper/issues/622 we found a big bug in the A11y View. It brought up a point about who is using this document, because that changes how strict we are in handling problems with it. I see it as a spectrum:

---less use---
* We use it to develop our sims with, maybe showing it to the IDRC sometimes, but for the most part it is just a tool for us (PhET A11y) to use.
* We use it, and may show it at presentations or in interviews, but we would never publish or share a link to it.
* We pass out a11y view links for spot/dev sims for some people to use, although it is *Public*, not many use it.
* We advertise the A11y View as a tool for all to use, and make available on our website for teaching/development.
---lots of use---

The more use, the more time that should be taken in a few areas
* design time to address pixel polishes/look and feel
* dev time to hammer out bugs
* qa time to find bugs
* qa time to test each release

@emily-phet I know that we have brought this up before, but It would be good to hammer out a more specific game plan to help address the issues that keep coming up. @terracoda @jessegreenberg as well @mbarlow12 feel free to add in. I'll probably bring this up at a meeting soon.

**Comment by zepumph** [comment]
For example, the scrolling feature doesn't work on IE, is this acceptable moving forward? Or should I spend more dev time to support that feature?

**Comment by terracoda** [comment]
@zepumph FYI, we did publish a link to it in the CSUN paper we wrote, but the published link was incorrect. I made a correction on the PhET Accessibility page.

One big improvement visually would be if the iframe on the left was fixed as the content on the right scrolled. Also, the fixed heights on the alerts are too small. I don't think a fixed height is the best approach to stop the bounciness.

**Comment by zepumph** [comment]
A few thoughts from a meeting regarding this issue:

What is the a11y view for?
@emily-phet: The accessibility content that we add to our simulations. Content/structure focused.
@terracoda: I see it as a way for visual users to experience what a screen reader user does.

We together decided that this is a document that deserves some love, and will be used in multiple places, and shown to others outside the project.

We came up with https://github.com/phetsims/a11y-research/issues/66 as a list of features that will push the a11y view to the next level, and help to convey our a11y content more effectively to all audiences that use the document. 

Thanks for a great meeting today @terracoda @emily-phet @jessegreenberg. Closing.
